babel.config.js

```diff
@@ -1,24 +1,31 @@
-module.exports = function (api) {
+module.exports = function(api) {
   api.cache(true);
 
   const presets = [
-    "@babel/preset-env",
-    "@babel/preset-react",
-    "@babel/preset-typescript"
+    '@babel/preset-env',
+    '@babel/preset-react',
+    '@babel/preset-typescript'
   ];
 
   const plugins = [
     '@babel/plugin-proposal-optional-chaining',
     [
       '@babel/plugin-transform-runtime',
       {
-        'corejs': 3
+        corejs: 3
       }
-    ]
+    ],
+    /**
+     * Add these options to silence the Storybook build warnings about different loose mode configuration
+     * @see https://github.com/babel/babel/issues/11622
```

Comment:
I think this should follow the jsdoc formatting for a URL:

`@see {@link https://github.com/babel/babel/issues/11622}`

It's only a minor point, but it should help with consistency across the codebase if we follow jsdoc formatting exactly. 

Also (I think) following the `@link` convention means jsdoc add-ons for our editor software will be able to use syntax highlighting for these comments, to make them a little more readable!

Reply:
@chriddell, I run these command on a different branch and this is what I was getting:

**npm audit fix:**
- updates only `package-lock.json`
- it changes `lockfileVersion` from 1 to 2
- `added 314 packages, removed 148 packages, changed 177 packages, and audited 3186 packages in 49s`

**npm audit fix --force:**
- updates `package-lock.json` and `package.json`
- `added 36 packages, removed 89 packages, changed 26 packages, and audited 3133 packages in 12s`